F/fzf/build_tarballs.jl

```diff
@@ -0,0 +1,35 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "fzf"
+version = v"0.21.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/junegunn/fzf.git", "4b518e50a3fce78eebff136dec2f1c8bee7c611c")
```

Comment:
What commit is that?  That revision gives me a 404 error: https://github.com/junegunn/fzf/commit/4b518e50a3fce78eebff136dec2f1c8bee7c611c.  The commit corresponding to v0.21.1 is https://github.com/junegunn/fzf/commit/334a4fa15920b08f7be2432398e620587cc6f079

Reply:
weird:
```
you will have to manually update the recorded commit.
> 0.21.1

Recorded as 4b518e50a3fce78eebff136dec2f1c8bee7c611c

Would you like to download additional sources?  [y/N]: N

			# Step 2b: Obtain binary dependencies (if any)

Do you require any (binary) dependencies?  [y/N]: N

[ Info: Updating bare Yggdrasil clone in deps/Yggdrasil...
Enter a name for this project.  This will be used for filenames:
> fzf

Enter a version number for this project:
> 0.21.1
```

Reply:
No idea what LibGit2 is doing, but that looks wrong to me:
```suggestion
    GitSource("https://github.com/junegunn/fzf.git", "334a4fa15920b08f7be2432398e620587cc6f079")
```

Reply:
Yeah, we have a problem with LibGit:
```
julia> obj = LibGit2.GitObject(repo, "0.21.1")
GitTag:
Tag name: 0.21.1 target: 334a4fa15920b08f7be2432398e620587cc6f079

julia> LibGit2.string(LibGit2.GitHash(obj))
"4b518e50a3fce78eebff136dec2f1c8bee7c611c"
```

Reply:
Oh, ok, Keno explained the problem on Slack: `4b518e50a3fce78eebff136dec2f1c8bee7c611c` is the hash of the tag object, `334a4fa15920b08f7be2432398e620587cc6f079` is the hash of the commit.  They both point to the same revision in the end, but `4b518e50a3fce78eebff136dec2f1c8bee7c611c` doesn't show up in the list of commits in GitHub.